lib/lmi_for_all.rb

```diff
@@ -0,0 +1,61 @@
+class LmiForAll
+  def initialize(soc_code)
+    @soc_code = soc_code
```

Comment:
:+1: currently refactoring this to the following structure:

`LmiForAll.new(lmi_client).lookup(soc_code)`